exe/floe

```diff
@@ -4,29 +4,39 @@
 require "floe"
 require "optimist"
 
+supported_docker_runners = {
+  "docker"     => Floe::Workflow::Runner::Docker,
+  "podman"     => Floe::Workflow::Runner::Podman,
+  "kubernetes" => Floe::Workflow::Runner::Kubernetes,
+  nil          => Floe::Workflow::Runner::Docker # default
+}
+
 opts = Optimist.options do
   version("v#{Floe::VERSION}\n")
+
   opt :workflow, "Path to your workflow json",                 :type => :string, :required => true
   opt :input, "JSON payload to input to the workflow",         :default => '{}'
   opt :credentials, "JSON payload with credentials",           :default => "{}"
-  opt :docker_runner, "Type of runner for docker images",      :default => "docker"
-  opt :docker_runner_options, "Options to pass to the runner", :type => :strings
+  opt :docker_runner, "Type of runner for docker images",      :type => :string, :short => 'r'
```

Comment:
The upside to keep the "default" here is that the help text shows it's the default.  Compare:

```diff
- -d, --docker-runner=<s>             Type of runner for docker images (default: docker)
+ -d, --docker-runner=<s>             Type of runner for docker images
```

Reply:
if docker-runner is defaulted, then we do not know if someone specifies docker with `--docker-runner=docker` (and we should ignore a `-k`) or if we should go with kubernetes because they did not specify a `docker-runner` and rather wanted